src/components/MoneyRequestConfirmationList.tsx

```diff
@@ -520,7 +520,8 @@ function MoneyRequestConfirmationList({
         if (updatedTagsString !== TransactionUtils.getTag(transaction) && updatedTagsString) {
             IOU.setMoneyRequestTag(transactionID, updatedTagsString);
         }
-    }, [policyTagLists, transaction, transactionID, policyTags, canUseViolations]);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I think it'll be better to document the reason why needs to have disable comments here. Yeah, I know that the original PR also didn't add the reason. Just a short reason documented will make it easier to read and maintain the code.

Reply:
Great idea! Please take a look and tell me what you think.